xmtp_mls/src/storage/encrypted_store/association_state.rs

```diff
@@ -62,4 +68,86 @@ impl StoredAssociationState {
             })
             .transpose()
     }
+
+    pub fn batch_read_from_cache(
```

Comment:
nit: this could be a `DbConnection` method

Reply:
Agreed. Rich put the other `StoredAssociationState` methods on the Struct itself so I decided to follow his pattern. But I agree we should make all three methods on the DbConnection to match the rest of the storage modules. Can do as a followup